doc/u-boot-env.txt

```diff
@@ -91,8 +91,8 @@ https://github.com/renesas-rcar/flash_writer/blob/rcar_gen3/docs/application-not
 
 NOTE 1
 Pay attention that tee.bin from the daily build is provided in two forms: with a prepended header and raw binary.
-In the command below, we use raw form which name starts with 'tee_raw-*'.
-File without '_raw' in the name can not be used for flashing as described below.
+In the command below, we use raw form which name starts with 'tee-raw-*'.
```

Comment:
As I see we still have underscore in name of this file in our product.
See https://github.com/xen-troops/meta-xt-images/pull/180/commits/415cab556a93ef9e00f99557ac658722c6a54ab0

Reply:
Hm, provided commit changes "_"  to "-" so do mine?

Reply:
The underscore in our build is because the mentioned patch not used(it is my guess).